packages/rocketchat-lib/server/lib/interceptDirectReplyEmails.js

```diff
@@ -0,0 +1,124 @@
+const IMAP = require('imap');
```

Comment:
Use the import format instead of requires (replace the others in other files too)
```javascript
import IMAP from 'imap';
```